src/Tempest/Database/src/IsDatabaseModel.php

```diff
@@ -73,7 +74,11 @@ public static function new(mixed ...$params): self
      */
     public static function query(): ModelQueryBuilder
     {
-        return new ModelQueryBuilder(self::class);
+        $queryBuilderAttribute = reflect(self::class)->getAttribute(QueryBuilder::class);
```

Comment:
Better to use `new ClassReflector` if we already know it's a class

Reply:
Yeah, thanks for your review. I'll update asap